### PR TITLE
fix whitespace in urls.

### PR DESCRIPTION
**Pull request by radiovisual**
I figured an example on creating relative urls should not have its urls breaking. :)

### CODE REVIEW

File: templates/page.html [hunk 1]
```diff
@@ -11,11 +11,11 @@
...
 
-        <p><a href="{{relative_path path " index.html "}}">Home</a></p>
-        <p><a href="{{relative_path path " services/marketing.html "}}">Services &gt; Marketing</a></p>
-        <p><a href="{{relative_path path " services/marketing/email.html "}}">Services &gt; Marketing &gt; Email</a></p>
+        <p><a href="{{relative_path path "index.html"}}">Home</a></p>
+        <p><a href="{{relative_path path "services/marketing.html"}}">Services &gt; Marketing</a></p>
+        <p><a href="{{relative_path path "services/marketing/email.html"}}">Services &gt; Marketing &gt; Email</a></p>
 
-        <script src="{{relative_path path " assets/js/bootstrap.min.js "}}"></script>
+        <script src="{{relative_path path "assets/js/bootstrap.min.js "}}"></script>
```
Comment (unstoppablecarl) [inline templates/page.html]: missed one

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 build
 node_modules
+.idea
```
Comment (unstoppablecarl) [inline .gitignore]: did you mean to add this?

**Comment by unstoppablecarl** [comment]
I'm not sure how those got in there strange.

File: .gitignore (hunk 2 above)
Comment (radiovisual) [inline .gitignore]: Actually, yes. I added it by mistake. I was working on another repo at the time, and this line made it into the wrong .gitignore. Oops! :confused: My IDE (Webstorm) loves to attach its _massive_ configuration file to the repo, so I add that line to all my .gitignore files.

Comment (radiovisual) [inline .gitignore]: I can remove it and push the change to this PR.

**Comment by radiovisual** [comment]
That should do it.

**Pull request merged**